### PR TITLE
Added userguid for recovery devices

### LINKED ISSUE #8826: User Guide: recovery files
The section [Manage your devices](https://docs.parsec.cloud/en/latest/userguide/manage_devices.html) should include instructions on how to recover access to Parsec via the recovery files.

### PR DESCRIPTION
**Pull request by Max-7**
Closes #8826 

Wait for #8742 

- [X] Keep changes in the pull request as small as possible
- [X] Ensure the commit history is sanitized
- [X] Give a meaningful title to your PR
- [X] Describe your changes
- [X] Link any related issue in the description

### CODE REVIEW

File: docs/userguide/manage_devices.rst [hunk 1]
```diff
@@ -53,3 +55,92 @@ You can add a new device from your profile page.
...
+The passphrase is used to unlock the file and get access to your data.
+
+.. note::
+
+  A recovery device is linked to a specific organization. If you have multiple organization,
+  you will have to create multiple recovery devices.
+
+.. caution::
+
+  If someone gets access to both the file and the passphrase, they can get
```
Comment (mmmarcos) [inline docs/userguide/manage_devices.rst:82]:
```suggestion
  If someone gets access to both the recovery file and the secret key, they can get
```

File: docs/userguide/manage_devices.rst [hunk 2]
```diff
@@ -53,3 +55,92 @@ You can add a new device from your profile page.
...
+    :width: 400
+    :alt: Create a recovery device
+
+Once you're ready, click on ``I understand``.
+
+.. image:: screens/export_recovery_device_page.png
+    :align: center
+    :alt: Export a recovery device
+
+You'll be able to download both the file and the passphrase. Make sure to get them both.
```
Comment (mmmarcos) [inline docs/userguide/manage_devices.rst:108]:
```suggestion
You'll be able to download both the Recovery File and the Secret Key. Make sure to get them both.
```

File: docs/userguide/manage_devices.rst [hunk 3]
```diff
@@ -53,3 +55,92 @@ You can add a new device from your profile page.
...
+  As mentioned before, these two files combined will allow someone to access your data.
+  Store them separately and don't let someone access them.
+
+.. image:: screens/export_recovery_device_download.png
+    :align: center
+    :alt: Download the recovery file and passphrase
+
+
+Import a recovery device
+------------------------
```
Comment (mmmarcos) [inline docs/userguide/manage_devices.rst:121]: I know this is the import action, but maybe we can rephrase it to be more user friendly. What do you think?
```suggestion
Use a recovery file to get access to your account
-------------------------------------------------
```
or
```suggestion
Use a recovery file to get back access
--------------------------------------
```

File: docs/userguide/manage_devices.rst [hunk 4]
```diff
@@ -53,3 +55,92 @@ You can add a new device from your profile page.
...
+
+If you forget your password and you have both the recovery file and the passphrase,
+you can import them and gain access to your data back.
```
Comment (mmmarcos) [inline docs/userguide/manage_devices.rst:124]:
```suggestion
If you forget your password and you have both the Recovery File and the Secret Key,
you can use them and gain access back.
```

File: docs/userguide/manage_devices.rst [hunk 5]
```diff
@@ -53,3 +55,92 @@ You can add a new device from your profile page.
...
+
+When trying to log in, click on ``Forgot your password?``.
+
+.. image:: screens/forgot_password.png
+    :align: center
+    :width: 350
+    :alt: Click on password forgotten
+
+You will be able to import the recovery file and type in the passphrase.
```
Comment (mmmarcos) [inline docs/userguide/manage_devices.rst:133]:
```suggestion
You will be able to import the Recovery File and type in the Secret Key.
```

File: docs/userguide/manage_devices.rst [hunk 6]
```diff
@@ -53,3 +55,92 @@ You can add a new device from your profile page.
...
+    :width: 400
+    :alt: Click on password forgotten
+
+.. image:: screens/import_recovery_device_filled.png
+    :align: center
+    :width: 400
+    :alt: Click on password forgotten
+
+
+Once imported, this will create a new device and you will be able to log into your organization by clicking ``Next``.
```
Comment (mmmarcos) [inline docs/userguide/manage_devices.rst:146]:
```suggestion
Once imported, Parsec will create a new device and you will be able to log into your organization by clicking ``Next``.
```

**Review by mmmarcos — COMMENTED** [review]
Honestly, I will stick with "Recovery files" (instead of "recovery device") everywhere regarding user: GUI, docs, etc. 
What do you think?

**Review by FirelightFlagboy — COMMENTED** [review]
Did you use the command `make intl-fr`, to (re)generate the po files ?

It seems the modified files are just because the wrapping have changed.
Maybe that was cause by a previous change that didn't use that command :thinking:

**Comment by Max-7** [comment]
> Did you use the command `make intl-fr`, to (re)generate the po files ?
> 
> It seems the modified files are just because the wrapping have changed. Maybe that was cause by a previous change that didn't use that command 🤔

Yeah I did. From experience, the generation of those files almost seem random sometimes, I had the same kind of problem with the v2 of Parsec. I swear you can launch the exact same command with the same config on two different computers and end up with different results -_-

**Review by mmmarcos — APPROVED** [review]
